### PR TITLE
ez contacts feedback

### PR DESCRIPTION
**Pull request by karloscastaneda**

### CODE REVIEW

File: static/carousel/css/styles.css [hunk 1]
```diff
@@ -47,7 +47,6 @@ body  {
   right: 0;
   position: absolute;
   margin: auto;
-  -webkit-transition: all .2s ease;
```
Comment (rfornes) [inline static/carousel/css/styles.css]: Just wondering if this was this intentional:
To remove `-webkit-` vendor prefix

I'm not sure but it's probably safe to remove it, please confirm, thanks.

File: layouts/partials/carousel/modal-content.css [hunk 2]
```diff
@@ -4,6 +4,8 @@
   {{ end }}
 {{ end }}
 
+"{{ partial (printf "%s/custom-fonts.css" .Type) . }}" +
```
Comment (rfornes) [inline layouts/partials/carousel/modal-content.css]: Why I can't see a `custom-fonts.css` partial in this PR files?
Need to know why this is needed, thanks.

File: static/carousel/css/styles.css (hunk 1 above)
Comment (karloscastaneda) [inline static/carousel/css/styles.css]: this was from running grunt, i don't know why it removed those, i'm taking a look

File: layouts/partials/carousel/modal-content.css (hunk 2 above)
Comment (karloscastaneda) [inline layouts/partials/carousel/modal-content.css]: i put the `custom-fonts` file on the root of the ez-contacts project partials

File: static/sidebar/js/modal/index.js [hunk 3]
```diff
@@ -54,6 +54,10 @@
         var holder = Utils.getByClass('tvp-products-holder');
```
Comment (rfornes) [inline static/sidebar/js/modal/index.js]: Please remove your changes related to ratings.
We will include this in a separate branch.

File: static/carousel/js/modal/index.js [hunk 4]
```diff
@@ -54,6 +54,10 @@
         var holder = Utils.getByClass('tvp-products-holder');
```
Comment (rfornes) [inline static/carousel/js/modal/index.js]: Please remove your changes related to ratings.
We will include this in a separate branch.

File: static/sidebar/gruntfile.js [hunk 5]
```diff
@@ -37,11 +40,11 @@ module.exports = function(grunt) {
         }
       },
       uglify: {
-        options: {
```
Comment (rfornes) [inline static/sidebar/gruntfile.js]: Please revert this change, we will check if this does not work for everyone first and if necessary we will need to find a different package to do this task

File: static/carousel/gruntfile.js [hunk 6]
```diff
@@ -38,11 +41,11 @@ module.exports = function(grunt) {
         }
       },
       uglify: {
-        options: {
-          compress: {
-            drop_console: true
-          }
-        },
```
Comment (rfornes) [inline static/carousel/gruntfile.js]: Please revert this change, we will check if this does not work for everyone first and if necessary we will need to find a different package to do this task
